test/robosherlock/test.cpp

```diff
@@ -0,0 +1,37 @@
+#include <string>
+#include <gtest/gtest.h>
+#include "rs/utils/RSAnalysisEngineManager.h"
+#include "rs/utils/RSAnalysisEngine.h"
+#include <rs/utils/common.h>
+#include <rs/types/all_types.h>
+#include <rs/scene_cas.h>
+
+#include <pcl/point_types.h>
+#include <ros/ros.h>
+
+class ExampleTest : public ::testing::Test
+{
+protected:
+    virtual void SetUp()
+    {
+        rs::common::getAEPaths("test",engineFile);
+        engine.init(engineFile);
+    }
+
+    virtual void TearDown(){}
+    std::string engineFile;
+    RSAnalysisEngine engine;
+
```

Comment:
I included the PipelineManager in the RSAnalysisEngine in the last changes I did.